src/components/directives/in-view/test/helpers.ts

```diff
@@ -0,0 +1,117 @@
+/*!
+ * V4Fire Client Core
+ * https://github.com/V4Fire/Client
+ *
+ * Released under the MIT license
+ * https://github.com/V4Fire/Client/blob/master/LICENSE
+ */
+
+import type { Locator, Page } from 'playwright';
+
+import type { Watcher, WatchHandler, WatchOptions } from 'core/dom/intersection-watcher';
+
+import { Component, Scroll } from 'tests/helpers';
+
+import { TEST_DIV_MARGIN_TOP_PX } from 'components/directives/in-view/test/const';
+
+/**
+ * A handler to pass to v-in-view
+ * @param watcher - The parameter of the watch handler
+ */
+function handler(watcher: Watcher): void {
+	const div = watcher.target;
+
+	const previousValue = parseInt(
+		div.getAttribute('data-test-in-view') ?? '0',
+		10
+	);
+
+	const nextValue = previousValue + 1;
+	watcher.target.setAttribute('data-test-in-view', nextValue.toString());
+}
+
+/**
+ * Returns the value of the watcher call counter stored in given locator.
+ * @param locator - The source locator
```

Comment:
После - с маленькой буквы

```suggestion
 * @param locator - the source locator
```